lxd/storage/drivers/driver_btrfs_volumes.go

```diff
@@ -774,14 +780,14 @@ func (d *btrfs) createVolumeFromMigrationOptimized(vol Volume, conn io.ReadWrite
 }
 
 // RefreshVolume provides same-pool volume and specific snapshots syncing functionality.
-func (d *btrfs) RefreshVolume(vol Volume, srcVol Volume, srcSnapshots []Volume, allowInconsistent bool, op *operations.Operation) error {
+func (d *btrfs) RefreshVolume(vol VolumeCopy, srcVol VolumeCopy, refreshSnapshots []Volume, allowInconsistent bool, op *operations.Operation) error {
```

Comment:
As srcVol will contain all of the snapshots and their configs, do we also need to pass some of the same data again in refreshSnapshots, can we instead just pass a slice of snapshot names? Is the config in refreshSnapshots somehow different to the snapshot configs in vol or srcVol?

Reply:
I agree and have just realized this, I'll replace it with a `[]string` in the next push.

Reply:
Otherwise it's just duplicate information.